src/lightning_app/storage/orchestrator.py

```diff
@@ -48,10 +48,10 @@ def __init__(
         self.response_queues = response_queues
         self.copy_request_queues = copy_request_queues
         self.copy_response_queues = copy_response_queues
-        self.waiting_for_response: Set[str] = set()
+        self.waiting_for_response: Dict[str, str] = {}
         self._validate_queues()
         self._exit_event = threading.Event()
-        self._sleep_time = 0.1
+        self._sleep_time = 5
```

Comment:
I think a sleep time of 5 seconds for storage updates is a bit too long. 2 seconds?

Reply:
Yes, I think both are fine. But moved to 2.